app/features/Contents/TradingSidebar/AssetAmount/useAssetAmount.test.ts

```diff
@@ -22,6 +23,7 @@ const createStore = (balance: number, exchangeRate: number) => {
   store.amm = {
     ...store.amm,
     price: exchangeRate,
```

Comment:
I do not believe it is necessary to adjust `price` for the particular tests you are doing, but I suppose you want to keep it here for the future, right?